### PR TITLE
Update Default Values for Limits

### PR DESCRIPTION
**Pull request by dubeejw**
- Display 256 as default memory limit
- Display 10 as default log limit
- Display 60000 as default timeout limit
- Fixes https://github.com/openwhisk/openwhisk/issues/1172

### CODE REVIEW

**Comment by dubeejw** [comment]
PG2 390.

File: tools/cli/go-whisk-cli/commands/action.go [hunk 1]
```diff
@@ -632,9 +645,9 @@ func init() {
   actionCreateCmd.Flags().BoolVar(&flags.action.sequence, "sequence", false, wski18n.T("treat ACTION as comma separated sequence of actions to invoke"))
   actionCreateCmd.Flags().StringVar(&flags.action.kind, "kind", "", wski18n.T("the `KIND` of the action runtime (example: swift:3, nodejs:6)"))
   actionCreateCmd.Flags().StringVar(&flags.action.shared, "shared", "no", wski18n.T("action visibility `SCOPE`; yes = shared, no = private"))
-  actionCreateCmd.Flags().IntVarP(&flags.action.timeout, "timeout", "t", -1, wski18n.T("the timeout `LIMIT` in milliseconds when the action will be terminated"))
-  actionCreateCmd.Flags().IntVarP(&flags.action.memory, "memory", "m", -1, wski18n.T("the memory `LIMIT` in MB of the container that runs the action"))
-  actionCreateCmd.Flags().IntVarP(&flags.action.logsize, "logsize", "l", -1, wski18n.T("the log size `LIMIT` in MB of the container that runs the action (default 10MB)"))
+  actionCreateCmd.Flags().IntVarP(&flags.action.timeout, "timeout", "t", TIMEOUT_LIMIT, wski18n.T("the timeout `LIMIT` in milliseconds when the action will be terminated"))
```
Comment (rabbah) [inline tools/cli/go-whisk-cli/commands/action.go]: thinking how we can improve this help message... what about: `timeout in milliseconds after which the action is terminated`.

File: tools/cli/go-whisk-cli/commands/action.go [hunk 2]
```diff
@@ -632,9 +645,9 @@ func init() {
...
+  actionCreateCmd.Flags().IntVarP(&flags.action.memory, "memory", "m", MEMORY_LIMIT, wski18n.T("the memory `LIMIT` in MB of the container that runs the action"))
```
Comment (rabbah) [inline tools/cli/go-whisk-cli/commands/action.go]: `the maximum allowed memory in MB for the action.`

File: tools/cli/go-whisk-cli/commands/action.go [hunk 3]
```diff
@@ -632,9 +645,9 @@ func init() {
...
+  actionCreateCmd.Flags().IntVarP(&flags.action.logsize, "logsize", "l", LOGSIZE_LIMIT, wski18n.T("the log size `LIMIT` in MB of the container that runs the action"))
```
Comment (rabbah) [inline tools/cli/go-whisk-cli/commands/action.go]: `the maximum log size in MB for the action.`?

**Comment by rabbah** [comment]
Verified - suggested some tweaks to the help message. What do you think?

**Comment by dubeejw** [comment]
Updated the messages based on your suggestions. We need to include `LIMIT` in the messages so that the help will show the an argument is expected after the flag. For instance, ```  -l, --logsize LIMIT          the maximum log size LIMIT in MB for the action (default 10)```

**Pull request merged**